extensions/podman/src/base-check.ts

```diff
@@ -21,11 +21,28 @@ export abstract class BaseCheck implements extensionApi.InstallCheck {
   abstract title: string;
   abstract execute(): Promise<extensionApi.CheckResult>;
 
-  protected createFailureResult(description?: string, title?: string, url?: string): extensionApi.CheckResult {
+  protected createFailureResult(
+    description?: string,
+    linksDescription?: string,
+    linkTitle?: string,
+    linkUrl?: string,
+    fixFailedCheckCommandId?: string,
+    fixFailedCheckCommandTitle?: string,
+  ): extensionApi.CheckResult {
```

Comment:
I think it's starting to become a nightmare with all these parameters

so probably need to switch to a JSON like object/interface

it'll be also more readable from client/consumer

instead of
```
this.createFailureResult(
          'To be able to run WSL2 you need Windows 10 Build 18362 or later.',
          'Learn about WSL requirements:',
          'WSL2 Install Manual',
          'https://docs.microsoft.com/en-us/windows/wsl/install-manual#step-2---check-requirements-for-running-wsl-2',
        );
```

if it's more like

```ts
this.createFailureResult({
description:  'To be able to run WSL2 you need Windows 10 Build 18362 or later.',
descriptionLink: 'Learn about WSL requirements:'
linkTitle: 'WSL2 Install Manual'
linkUrl: 'https://docs.microsoft.com/en-us/windows/wsl/install-manual#step-2---check-requirements-for-running-wsl-2'
});
```